docs/develop/datasets.rst

```diff
@@ -21,3 +21,58 @@ While a part of the MONETIO repository,
 the private MELODIES MONET readers are designated with prefix ``_mm``.
 
 Support for additional models is also under developed.
+
+Standard variables are required to be computed in each model reader for each capability including surface, aircraft, and satellite as specified in the table below.
+
+.. list-table:: Required Variables for Model Readers
+   :widths: 10 30 30 30
+   :header-rows: 1
+
+   * - Capability
+     - | Variable Name 
+       | in Code
+     - Description
+     - Additional Requirements
+   * - Surface
+     - | ``time``
+       | ``latitude``
+       | ``longitude``
+     - | Time in ``datetime64[ns]`` format
+       | Latitude in degrees
+       | Longitude in degrees
+     - | Provide only surface model data 
+       | or if provide vertical model data, 
+       | first level must be the level 
+       | nearest to the surface.
+       | All gases are in ppb and 
+       | all aerosols are in µg/m3.
```

Comment:
Not sure if this is a "requirement", as you can get around magnitude differences, e.g., using MM config. But certainly it's usually easier if they are in these units.

Reply:
I want to list it as a requirement, so that there is consistency in the tool. But of course, if people prefer not to do this, there will not be an error and you can fix it in the unit conversion step later.

Reply:
I would also prefer "ppbv" if you agree.